### PR TITLE
Related #6385.

### PR DESCRIPTION
**Pull request by kernehed**
Configuration variable explanation style update.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/camera.local_file.markdown [hunk 1]
```diff
@@ -27,9 +27,16 @@ camera:
...
 
 Configuration variables:
-
- - **file_path** (*Required*): File to serve as the camera.
- - **name** (*Optional*): Name of the camera
+{% configuration %}
+file_path:
+  description: File to serve as the camera.
+  required: true
+  type: init
```
Comment (frenck) [inline source/_components/camera.local_file.markdown]: `init` is not a valid type.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by kernehed — COMMENTED** [review]
This should now be fixed.

File: source/_components/camera.local_file.markdown [hunk 2]
```diff
@@ -27,9 +27,16 @@ camera:
 ```
 
 Configuration variables:
```
Comment (frenck) [inline source/_components/camera.local_file.markdown]: Please remove this duplicate header...

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @kernehed!

Very nice first-time contribution to our documentation! 🥇 
Feel free to submit more PR's 😉 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**